### DISCUSSION TITLE
Handles Programming Guide Doesn't Exist

### DISCUSSION THREAD

**Discussion by eccentricdevotion** [eccentricdevotion/TARDIS#478] [Wiki]
_Transferred from Bukkit Dev
The_RetroGade created this issue Mar 28, 2020_

Please update the site map to include it. Thanks!

**Comment by Chew** [comment]
Sounds like a job for me

**Comment by eccentricdevotion** [comment]
The biggest problem with this one is the lack of testing, I haven't really tried using the system after (half-heartedly) implementing it, as I've never really had need of the feature, and I'm not sure I actually finished it. I think you can create Handles program disks, but I'm not sure any/all features (such as events) are actually processed...
